fiftyone/core/collections.py

```diff
@@ -9051,13 +9051,26 @@ def list_indexes(self):
         """
         return list(self.get_index_information().keys())
 
-    def get_index_information(self, include_size=False):
+    def _sample_stats(self) -> dict:
+        """Returns the collection stats for the sample collection."""
```

Comment:
Private methods don't "require" docstrings. Could also move to simple utils method